src/dmd/globals.d

```diff
@@ -335,6 +335,8 @@ extern (C++) struct Global
 
     enum recursionLimit = 500; /// number of recursive template expansions before abort
 
+    extern (D) FileName function(FileName) preprocess;
```

Comment:
Having the function `extern(D)` is not much use for anyone overriding this.

Reply:
I have no idea what you need for an interface or what your plans are for it, so I will leave that up to you.